ibi/coreos-installer/Makefile

```diff
@@ -10,6 +10,7 @@ TMPDIR=tmp
 RHCOS_LIVE_ISO_URL=https://mirror.openshift.com/pub/openshift-v4/amd64/dependencies/rhcos/latest/rhcos-live.x86_64.iso
 RHCOS_LIVE_ISO=$(TMPDIR)/rhcos-live.iso
 COREOS_INSTALLER=quay.io/coreos/coreos-installer:latest
+LCA_IMAGE := $(or $(LCA_IMAGE), quay.io/openshift-kni/lifecycle-agent-operator:4.14.0)
```

Comment:
you can use
```
LCA_IMAGE ?= quay.io/openshift-kni/lifecycle-agent-operator:4.14.0
```

I think it is the same as the or clause that you used, just simpler to write and read